examples/export-data/main.ts

```diff
@@ -1,2 +1,183 @@
-// Run this with yarn export-data or npx tsx main.ts
-console.log("Hello World")
\ No newline at end of file
+import mongoose from 'mongoose';
+import dotenv from 'dotenv';
+import { dbConnect } from './utils/db-connect';
+import { AnalyticsViewer, AnalyticsLogger, AnalyticsManager } from "bog-analytics";
+import { ClickEventModel } from './models/click-event';
+import { VisitEventModel } from './models/visit-event';
+import { InputEventModel } from './models/input-event';
+import { CustomEventModel } from './models/custom-event';
+import { Project } from './utils/types';
+import { EventEnvironment } from './utils/types';
+import { getProjectIDByName, createProject } from './actions/project'
+
+dotenv.config();
+const CLIENT_KEY = process.env.CLIENT_API_SECRET || '';
+const SERVER_KEY = process.env.SERVER_API_SECRET || '';
+const projectName = "Bits of Good Example Project";
+
+const main = async () => {
+    try {
+        await dbConnect();
+        console.log('Connected to MongoDB');
+
+        // Create the project if it doesn't exist
+        let projectId = await getProjectIDByName(projectName);
+        if (!projectId) {
+            console.log(`Project '${projectName}' does not exist. Creating new project`);
+            
+            const newProject: Partial<Project> = {
+                projectName: projectName,
+                clientApiKey: CLIENT_KEY,
+                serverApiKey: SERVER_KEY,
+                privateData: false,
+                deletionPolicy: -1,
+            };
+            
+            const createdProject = await createProject(newProject);
+            projectId = createdProject._id;
+            console.log(`Created new project with ID: ${projectId}`);
+        } else {
+            console.log(`Project ID found: ${projectId}`);
+        }
```

Comment:
Let's remove this because the create project functionality doesn't really make sense in the context of a single project (i.e., all the events they retrieve will be for 1 project anyways)

this means when you are creating the events below, you don't need to specify a project id (since its not particularly relevant to the actual event data)

Reply:
you'll need to update the base event mongoose models so that they no longer have a project id associated with them